docs/1.0. Certificate Provisioning.md

```diff
@@ -206,6 +206,17 @@ An EST Client MUST provide a method to replace the manufacturer-issued TLS Clien
 
 An EST Client MAY ignore all timestamps in the certificate validity periods during the certificate provisioning process if it does not accurately know the current time, see [Bootstrapping Remote Secure Key Infrastructures - Section 2.6.1](https://tools.ietf.org/html/draft-ietf-anima-bootstrapping-keyinfra-41#section-2.6.1) for more information.
 
+If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD advertise that it supports BCP-003-03 using the 'tags' attribute of the Node API's `/self` endpoint. Control interfaces can identify all Nodes which support BCP-003-03 by a URN key 'urn:x-nmos:tag:certprov/', followed by the supported version of this BCP. When EST is enabled the value is set to `["enabled"]` and when disabled `["disabled"]`.
+
+**Example:** The 'tags' attribute of a NMOS Node
+```json
+...
+"tags": {
+  "urn:x-nmos:tag:certprov/v1.0": ["enabled"]
```

Comment:
:thought_balloon: Might this be reasonable? keep one tag that has a list of supported versions, similar to the api pages?

```suggestion
  "urn:x-nmos:tag:certprov": ["v1.0"]
```

Reply:
> The only downside I see is that didn't someone ask about distinguishing between support for the spec and that support being currently active/enabled?

I would explicitly say listed versions are active, if the tag or any version is missing then it is unsupported/disabled. Another talking point for the next call?

Reply:
Your proposal makes perfect sense to me.

Reply:
Yes, that makes sense to me too. We just need to make sure the text is clear that this shouldn't be present if the EST support is currently disabled by configuration.

Reply:
Sounds good, I have updated the docs to reflect this